03OnLearningAndTraining/README.md

```diff
@@ -332,31 +332,31 @@ In the following part, we will focus mainly on the first aspect \(preparation\)
+Η χρήση αρχών της ενεργής μάθησης και η εφαρμογή στην κατάρτιση είναι, γενικά, μια καλή ιδέα. Είστε ο δεύτερος καλύτερος κριτής για τα οφέλη της. Θυμηθείτε ότι ο πρώτος κριτής είναι ο συμμετέχων.
 
-Active learning helps to bypass diversity in learning styles and other difficulties with audiences. While more efficient in reaching outcomes of higher levels, active learning also addresses cognition issues related to the nature of the content and the way to present it, as shown in the following diagram, commonly found in several textbooks and online resources, and known as the Cone of Learning. Active learning is best utilised at the top levels of Bloom’s Taxonomy \(Analize, Define, Create, Evaluate\), and that also corresponds to the best strata of memorization: what you say, write or do - the bottom half of the Cone of Learning. Cognition issues arise with more ease when content spans several of these levels at a time and fails to address the intermediate levels as well. Checking your content against the Cone of Learning is an easy way of detecting these potential miss-outs while you deliver training. Likewise it allows you to decide to use more visual aids where you expect that the need for memorization is higher. So, when your audience gets behind you may use this technique to diagnose, try to locate the causes and pick the most effective remediation.
+Η Ενεργή Μάθηση συμβάλλει στην παράκαμψη της διαφορετικότητας των μορφών μάθησης και άλλων δυσκολιών με το κοινό. Ενώ είναι πιο αποτελεσματική στην επίτευξη αποτελεσμάτων υψηλότερων επιπέδων, η ενεργή μάθηση ασχολείται επίσης με γνωστικά ζητήματα σχετικά με τη φύση του περιεχομένου και τον τρόπο παρουσίασής του, όπως φαίνεται στο ακόλουθο διάγραμμα, το οποίο συνήθως βρίσκεται σε διάφορα εγχειρίδια και σε διαδικτυακούς πόρους, και είναι γνωστό ως Κώνος της Μάθησης. Η ενεργή μάθηση χρησιμοποιείται καλύτερα στα υψηλά επίπεδα της ταξινόμησης του Bloom \(Ανάλυση, Ορισμός, Δημιουργία, Αξιολόγηση\) και αντιστοιχεί επίσης στα καλύτερα στρώματα \(strata\) της απομνημόνευσης: ότι λέτε, γράφετε ή κάνετε - το κάτω μισό του Κώνου της Μάθησης. Τα γνωστικά θέματα προκύπτουν με μεγαλύτερη ευκολία όταν το περιεχόμενο εκτείνεται σε πολλά από αυτά τα επίπεδα κάθε φορά και δεν ανταποκρίνεται στα ενδιάμεσα επίπεδα. Ο έλεγχος του περιεχομένου σας σε σχέση με τον Κώνο της Μάθησης είναι ένας εύκολος τρόπος ανίχνευσης αυτών των πιθανών αδυναμιών κατά τη διάρκεια της εκπαίδευσης. Ομοίως, σας επιτρέπει να αποφασίσετε να χρησιμοποιήσετε περισσότερα οπτικά βοηθήματα όπου αναμένετε ότι η ανάγκη για απομνημόνευση είναι υψηλότερη. Έτσι, όταν το ακροατήριό σας υπολείπεται της διδασκαλίας σας μπορείτε να χρησιμοποιήσετε αυτήν την τεχνική για τη διάγνωση, ώστε προσπαθήσετε να εντοπίσετε τις αιτίες και να επιλέξετε την πιο αποτελεσματική μέθοδο διόρθωσης της κατάστασης.
```

Comment:
Missing diagram Cone of Learning. Is it https://github.com/Open-Science-Training-Handbook/Open-Science-TrainingHandbook_EL/blob/master/Images/image_5.png  ?. If yes it also needs translation before adding it to the Greek version

Reply:
Thanks for the translated image @vxdaskalou . I've created a new PR #18 just for the image.